projects/core/src/lib/components/header/header.component.html

```diff
@@ -0,0 +1,45 @@
+<header class="jsmu-header" [ngClass]="{mobile: sideBar}">
+	<nav class="jsmu-header__navigation">
```

Comment:
why don't you reuse jsmu-navigation component? please fix

Reply:
done